package.json

```diff
@@ -19,6 +19,15 @@
   },
```

Comment:
- I can't run the server kindly add the script into your ``json`` file.
![Screen Shot 2021-09-29 at 18 54 06](https://user-images.githubusercontent.com/22258274/135364056-46a3cb0e-2f93-4213-a192-c3bc98b43131.png)